libs/platform/core/src/services/context/context.controller.ts

```diff
@@ -37,6 +38,10 @@ export class ContextController implements ReactiveController {
       ),
     ]).pipe(
       skip(1),
+      tap(([overrideContext, context]) => {
```

Comment:
we should have unit tests for this behavior, also maybe @supproduction can take a look, if we don't break some edge cases (when we already have context provided on the host + override for example)